### PR TITLE
disjoint smart graphs changelog

### PR DESCRIPTION
**Pull request by hkernbach**
Only the changelog entry for disjoint smart graphs

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,15 @@
 devel
 -----
 
+* Added feature: DisjointSmartGraphs
+
+  SmartGraphs have been extended to a new subtype, called **DisjointSmartGraphs**.
```
Comment (markuspf) [inline CHANGELOG:6]: is `DisjointSmartGra` the official spelling?

File: CHANGELOG [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+        A DisjointSmartGraph prohibits connected edges between different
```
Comment (markuspf) [inline CHANGELOG:7]:
```suggestion
        A DisjointSmartGraph prohibits edges between different
```

File: CHANGELOG [hunk 3]
```diff
@@ -1,6 +1,15 @@
 devel
 -----
 
+* Added feature: DisjointSmartGraphs
+
+  SmartGraphs have been extended to a new subtype, called **DisjointSmartGraphs**.
+        A DisjointSmartGraph prohibits edges between different
+        SmartGraph components. In case the graph schema is able to be represented
```
Comment (markuspf) [inline CHANGELOG:8]:
```suggestion
        SmartGraph components. In case the graph schema can be represented
```

File: CHANGELOG (hunk 1 above)
Comment (hkernbach) [reply CHANGELOG:6]: Yes

**Pull request merged**